### PR TITLE
--update: using form state

### PR DESCRIPTION
**Pull request by mayanksh99**

### CODE REVIEW

File: src/components/Team/UpdateProfile.js [hunk 1]
```diff
@@ -153,23 +135,53 @@ const UpdateProfile = props => {
...
-					formData.append("dob", dob);
-					formData.append("github", github);
-					formData.append("linkedin", linkedin);
-					formData.append("twitter", twitter);
-					formData.append("portfolio", portfolio);
 
 					const res = await updateUserService(formData);
 					if (res.message === "success") {
 						_notification("success", "Success", "Event Updated");
+						if (newName !== null || newEmail !== null) {
```
Comment (rhnmht30) [inline src/components/Team/UpdateProfile.js:181]: This is not required on the frontend. User token expires when they change their name and email. They will get a session expired message when they try to access authenticated material. You need to catch that in the interceptor.

File: src/components/Team/UpdateProfile.js [hunk 2]
```diff
@@ -24,22 +24,13 @@ const UploadContainer = styled.div`
 
 const UpdateProfile = props => {
 	const [user, setUser] = useState(null);
-	const [name, setName] = useState(null);
 	const [isLoading, setIsLoading] = useState(false);
-	const [email, setEmail] = useState(null);
-	const [password, setPassword] = useState(null);
+	const [newName, setNewName] = useState(null);
```
Comment (rhnmht30) [inline src/components/Team/UpdateProfile.js:28]: These are not required in the component state. Name and email changes are already being handled on the backend.

File: src/components/Team/UpdateProfile.js [hunk 3]
```diff
@@ -228,15 +242,15 @@ const UpdateProfile = props => {
...
+										required: true,
 										message: "Please enter your name!"
 									}
 								]
 							})(
 								<Input
 									type="text"
 									placeholder="Name"
-									onChange={e => setName(e.target.value)}
+									onChange={e => setNewName(e.target.value)}
```
Comment (rhnmht30) [inline src/components/Team/UpdateProfile.js:253]: using ```onChange``` in these antd components is a wrong way to listen to changes. They will lose their values during state updates. Antd handles the listening for change events for us. You just need to grab the final values during submission.

**Comment by mayanksh99** [comment]
okay

**Pull request merged**